src/core_codemods/docs/pixee_python_combine-startswith-endswith.md

```diff
@@ -0,0 +1,11 @@
+This codemod updates places where two separate calls to either `startswith` or `endswith` are joined by an `or` statement. 
```

Comment:
```suggestion
Many developers are not necessarily aware that the `startswith` and `endswith` method of `str` objects can accept a tuple of strings to match. This means that there is a lot of code that uses boolean expressions such as `x.startswith('foo') or x.startswith('bar')` instead of the simpler expression `x.startswith(('foo', 'bar'))`.

This codemod simplifies the boolean expressions where possible which leads to cleaner and more concise code.
```